File: fizz-buzz-2.cpp
#include <iostream>
using namespace std;

int main() {
  int N;
  cin >> N;
  for (int i = 1; i <= N; ++i) {
    i % 3 == 0 && i % 5 == 0
    ? cout << "FizzBuzz" << endl
    : i % 5 == 0
      ? cout << "Buzz" << endl
      : i % 3 == 0
        ? cout << "Fizz" << endl
        : cout << i << endl;
  }
}

File: python-code/fizz-buzz-2.py
def main():
    N = int(input())
    for i in range(1, N+1):
        if i % 3 == 0 and i % 5 == 0:
            print("FizzBuzz")
        elif i % 5 == 0:
            print("Buzz")
        elif i % 3 == 0:
            print("Fizz")
        else:
            print(i)

if __name__ == '__main__':
    main()
